pkg/evaluator/evaluator.go

```diff
@@ -126,6 +135,36 @@ func (e *Evaluator) evalReturn(scope *scope, ret *parser.Return) Value {
 	return &ReturnValue{Val: val}
 }
 
+func (e *Evaluator) evalIf(scope *scope, i *parser.If) Value {
+	val := e.evalCoditionedBlock(scope, i.IfBlock)
```

Comment:
:pineapple: Thinking out aloud here: The AST for `If` could just have a single slice of condition blocks (i.e. no `if` or `else` parts). The `if` bit is just the first of the conditional blocks, and the `else` could just have a condition of `TRUE`. This would reduce the evaluation to just looping over that one slice and evaluating it all the same way.

Reply:
yeah, as discussed in person - this is a nice and accurate observation and will lead to less code :+1:, but it might be a bit confusing so for now I'll leave as is.